tests/fixtures.py

```diff
@@ -103,6 +103,10 @@ def get_node_thread_pool_search_queue_size(self):
             nodes = self.get('/_nodes?filter_path=**.thread_pool').json()['nodes'].values()
             return [node['settings']['thread_pool']['search']['queue_size'] for node in nodes]
 
+        def get_processors_setting(self):
+            nodes = self.get('/_nodes/settings?filter_path=**.processors').json()['nodes'].values()
+            return [node['settings']['processors'] for node in nodes]
```

Comment:
I don't think this backports cleanly to 5.6:

```

self = <tests.fixtures.elasticsearch.<locals>.Elasticsearch object at 0x7f937edd2d30>

    def get_processors_setting(self):
>       nodes = self.get('/_nodes/settings?filter_path=**.processors').json()['nodes'].values()
E       KeyError: 'nodes'

tests/fixtures.py:82: KeyError
```

Is there a different path, off the top of your head?

Reply:
It should be the same. 😕 

You ran into this after backporting the main change into `es-docker` (different than in `docker-entrypoint.sh` as in this PR)?

Reply:
Ah, thank you.

I messed up the backport for the entrypoint script. Thank you for the test (it caught _my_ error), and sorry for the noise.

Reply:
Cheers!

Reply:
@jarpy Will you backport this to 6.4 too?

Reply:
Yes indeed. Testing locally now.

Reply:
Awesome! Thank you so much for that.